java-logging-dropwizard/build.gradle

```diff
@@ -1,52 +1,54 @@
 publishing {
-    publications {
-        Dropwizard(MavenPublication) {
-            from components.java
-            artifact sourcesJar
-            artifact javadocJar
-            groupId project.group
-            artifactId 'java-logging-dropwizard'
-            version project.version
-        }
+  publications {
+    Dropwizard(MavenPublication) {
+      from components.java
+      artifact sourcesJar
+      artifact javadocJar
+      groupId project.group
+      artifactId 'java-logging-dropwizard'
+      version project.version
     }
+  }
 }
 
 bintray {
-    user = System.getenv('BINTRAY_USER')
-    key = System.getenv('BINTRAY_KEY')
-    publications = ['Dropwizard']
-    publish = true
-    pkg {
-        repo = 'hmcts-maven'
-        name = 'java-logging-dropwizard'
-        userOrg = 'hmcts'
-        licenses = ['MIT']
-        vcsUrl = 'https://github.com/hmcts/java-logging'
-        version {
-            name = project.version
-        }
+  user = System.getenv('BINTRAY_USER')
+  key = System.getenv('BINTRAY_KEY')
+  publications = ['Dropwizard']
+  publish = true
+  pkg {
+    repo = 'hmcts-maven'
+    name = 'java-logging-dropwizard'
+    userOrg = 'hmcts'
+    licenses = ['MIT']
+    vcsUrl = 'https://github.com/hmcts/java-logging'
+    version {
+      name = project.version
     }
+  }
 }
 
 def versions = [
-        dropwizard: '1.0.5'
+  dropwizard: '1.0.5'
 ]
 
 dependencies {
-    compile group: 'io.dropwizard', name: 'dropwizard-core', version: versions.dropwizard
+  compile group: 'io.dropwizard', name: 'dropwizard-core', version: versions.dropwizard
+
+  testCompile group: 'junit', name: 'junit', version: '4.12'
```

Comment:
did it work before??

Reply:
ah, i think it's a leftover. had a dummy unit test before